Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php

```diff
@@ -18,9 +19,11 @@ class BlogController extends ActionController
      * $blog is a not yet persisted clone of the original blog containing
      * the modifications
      *
-     * @Validate(param="blog", validator="FriendsOfTYPO3\BlogExample\Domain\Validator\BlogValidator")
      * @throws NoBlogAdminAccessException
      */
+    #[Validate([
+        'validator' => BlogValidator::class,
+    ])]
```

Comment:
Found some solutions in DummyController of TYPO3 test sources:

```suggestion
    #[Validate([
        'param' => 'blog',
        'validator' => BlogValidator::class,
    ])]
```